src/main/plugins/org.dita.html5/xsl/toc.xsl

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file is part of the DITA Open Toolkit project.
+
+Copyright 2014 Jarno Elovirta
+
+See the accompanying LICENSE file for applicable license.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot"
+                xmlns:ditamsg="http://dita-ot.sourceforge.net/ns/200704/ditamsg"
+                version="2.0"
+                exclude-result-prefixes="xs dita-ot ditamsg">
+
+  <xsl:import href="plugin:org.dita.html5:xsl/dita2html5Impl.xsl"/>
```

Comment:
👎 for creating a deeper import structure. This import should be part of `map2html5-coverImpl_template.xsl` imports.